frontend/src/FullPageRoutes/ContactUs.tsx

```diff
@@ -51,12 +52,17 @@ export default function ContactUs(): JSX.Element {
   const [values, setValues] = useState(contactValue)
   const classes = useStyles()
 
-  const setContactValue = (e: any) => {
```

Comment:
why did you get rid of the set function? We will need both setContactValue and handleSubmit

Reply:
So just put the return request in the handleSubmit.

Reply:
right, you don't want to make the request until the user selects the submit button, but setContactValue should be called in the onChange of each input